### ISSUE TITLE
how can i set it up for myself

### ISSUE THREAD

**Issue by xxspobbixx** [sb-child/gitea-theme-github-dark#1]
Hello,

I would like to use this theme but I have Gitea running on Docker, can you help me?

Best regards

Florian

**Comment by sb-child** [comment]
can you access the `data` volume of your instance? you can modify your data`/custom/conf/app.ini`, and put css file in data`/custom/public/css/theme-ghdark.css`

**Comment by xxspobbixx** [comment]
i had to create the folders /custom/conf/ and /custom/public/css/ gitea also recognizes the theme after editing the app.ini, but unfortunately it doesn't load it when i activate it, the page is white

**Comment by sb-child** [comment]
> the page is white

Can you try opening the developer window, and check the css file is loaded?

maybe gitea can't found the css file.

UPDATE: check these docs https://docs.gitea.io/en-us/install-with-docker/#customization and https://docs.gitea.io/en-us/customizing-gitea/#customizing-the-look-of-gitea

**Comment by revoio** [comment]
you must insert CSS file in:
custom/public/```assets```/css/

**Comment by pscriptos** [comment]
I have been trying to configure a theme in my Gitea for a long time now.
However, I have the same problem as the user above me.
I also use Gitea in a Docker.
I have now tried to create folders but the .css file was never found.
As you can see, I have tried in different places.
In my `app.ini` the configuration is correct, the themes are offered for me to change.
However, the page turns white when I try to change the theme.

The permissions are set correctly, just like for other files.

```bash
➜  app tree gitea
gitea
├── assets
│   └── css
│       └── theme-ghdark.css
├── conf
│   └── app.ini
├── css
│   └── theme-ghdark.css
├── custom
│   └── public
│       ├── assets
│       │   └── css
│       │       ├── theme-carbonred.css
│       │       └── theme-ghdark.css
│       └── css
│           ├── theme-carbonred.css
│           └── theme-ghdark.css
├── public
│   ├── assets
│   │   └── css
│   │       └── theme-ghdark.css
│   └── css
│       └── theme-ghdark.css
├
```
datadir is here:
`/docker/volumes/gitea/app/gitea`

Can anyone support me?

Thank you, pscriptos

**Comment by sb-child** [comment]
can you check the `$GITEA_CUSTOM` variable was set to?

**Comment by pscriptos** [comment]
I have tried, but I get an error message when I try to do this within a docker. If this is the wrong way, then unfortunately I don't know how it works.

**Comment by pscriptos** [comment]
Guys, I have to take it all back.
This morning I tested and researched...
Then I went to the doctor and when I got back home, the theme was suddenly active. Without me having done anything.
Then I switched to another theme and it didn't work again.
Then I had the brilliant idea: I have a ReverseProxy and as stupid as I am, I was sure to activate the "cache assets" option.
Unfortunately, that's exactly what happened. 🙄
I deactivated the option and suddenly it worked.

I'm sorry that I've disturbed you with this topic.

Finally, I show my configuration here:

My Docker-Compose.yaml
```
version: '3.3'
services:
  app:
    image: gitea/gitea:1.21.9
    environment:
      - ...
    volumes:
      - /docker/volumes/gitea/app:/data
```

The folder in which I put my themes:
```
➜  / tree /docker/volumes/gitea/app/gitea/public
/docker/volumes/gitea/app/gitea/public
└── assets
    └── css
        ├── theme-carbonred.css
        └── theme-ghdark.css

2 directories, 2 files
```

Sorry and Thanks.
pscriptos
